lib/bolt/pal.rb

```diff
@@ -242,6 +242,7 @@ def list_modulepath
     end
 
     def parse_params(type, object_name, params)
+      return params if params.empty?
```

Comment:
With the change you added in CLI is it possible to have an empty params hash? It seems like by setting 'options[:params_parsed]' we don't actually parse_params? Maybe this is possible in testing though. Also would a test asserting that pal.parse_params is not invoked when it is not necessary appropriate?

Reply:
Whoops, I think I fixed this bug in two places. I was going to have this method return early then decided to make the CLI handle it instead. So this line can probably just be removed.